crates/typst/src/layout/page.rs

```diff
@@ -189,8 +189,11 @@ pub struct PageElem {
     ///
     /// When set to `{none}`, the background becomes transparent. Note that PDF
     /// pages will still appear with a (usually white) background in viewers,
-    /// but they are conceptually transparent. (If you print them, no color is
-    /// used for the background.)
+    /// but they are actually transparent. (If you print them, no color is used
+    /// for the background.)
+    ///
+    /// If you select white explicitly, will it physically color it in white? If
```

Comment:
I don't get what you find misleading here.

Reply:
If you specify white, and the printer can print white, it will indeed use that white.

Reply:
![Mind blown](https://github.com/user-attachments/assets/b2567f33-f3cf-4e37-8bb5-ba40f465d4a4)

But do you know any printer that can print white? Because if it's just a theoretical thing and not practical, then this argument doesn't hold.

Reply:
There are printers that can do this.

Reply:
TIL.